redis/src/main/scala/zio/redis/api/Keys.scala

```diff
@@ -13,72 +13,293 @@ import zio.{ Chunk, ZIO }
+    absTtl: Option[AbsTtl] = None,
+    idleTime: Option[IdleTime] = None,
+    freq: Option[Freq] = None
+  ): ZIO[RedisExecutor, RedisError, Unit] = Restore.run((key, ttl, value, replace, absTtl, idleTime, freq))
+
+  /**
+   * Iterates the set of keys in the currently selected Redis database. An iteration starts when the cursor is set to 0,
+   * and terminates when the cursor returned by the server is 0.
+   *
+   * @param cursor cursor value, starts with zero
+   * @param pattern key pattern
+   * @param count count option, specifies number of returned elements per call
+   * @param `type` type option, filter to only return objects that match a given type
+   * @return returns an updated cursor that the user needs to use as the cursor argument in the next call along with the values
+   *
+   * @see <a href="https://redis.io/commands/scan">https://redis.io/commands/scan</a>
+   */
   final def scan(
-    a: Long,
-    b: Option[Regex] = None,
-    c: Option[Long] = None,
-    d: Option[String] = None
-  ): ZIO[RedisExecutor, RedisError, (Long, Chunk[String])] = Scan.run((a, b, c, d))
-
-  final def touch(a: String, as: String*): ZIO[RedisExecutor, RedisError, Long] = Touch.run((a, as.toList))
-
-  final def ttl(a: String): ZIO[RedisExecutor, RedisError, Duration] = Ttl.run(a)
-
-  final def typeOf(a: String): ZIO[RedisExecutor, RedisError, RedisType] = TypeOf.run(a)
-
-  final def unlink(a: String, as: String*): ZIO[RedisExecutor, RedisError, Long] = Unlink.run((a, as.toList))
-
-  final def wait_(a: Long, b: Long): ZIO[RedisExecutor, RedisError, Long] = Wait.run((a, b))
+    cursor: Long,
+    pattern: Option[Regex] = None,
+    count: Option[Long] = None,
+    `type`: Option[String] = None
+  ): ZIO[RedisExecutor, RedisError, (Long, Chunk[String])] = Scan.run((cursor, pattern, count, `type`))
+
+  /**
+   * Alters the last access time of a key(s). A key is ignored if it does not exist.
+   *
+   * @param key one required key
+   * @param keys maybe rest of the keys
+   * @return The number of keys that were touched.
+   * @see <a href="https://redis.io/commands/touch">https://redis.io/commands/touch</a>
+   */
+  final def touch(key: String, keys: String*): ZIO[RedisExecutor, RedisError, Long] = Touch.run((key, keys.toList))
+
+  /**
+   * Returns the remaining time to live of a key that has a timeout.
+   *
+   * @param key key
+   * @return remaining time to live of a key that has a timeout, error otherwise
+   *
+   * @see <a href="https://redis.io/commands/ttl">https://redis.io/commands/ttl</a>
+   */
+  final def ttl(key: String): ZIO[RedisExecutor, RedisError, Duration] = Ttl.run(key)
+
+  /**
+   * Returns the string representation of the type of the value stored at key.
+   *
+   * @param key key
+   * @return type of the value stored at key
+   *
+   * @see <a href="https://redis.io/commands/type">https://redis.io/commands/type</a>
+   */
+  final def typeOf(key: String): ZIO[RedisExecutor, RedisError, RedisType] = TypeOf.run(key)
```

Comment:
[doc](https://redis.io/commands/type) says there's a `stream` type, which is somehow  missing

Reply:
That's added since the initial version was implemented. We can file this as a separate issue. Wdyt?